helper-cli/src/main/kotlin/commands/ListPackagesCommand.kt

```diff
@@ -49,6 +52,11 @@ class ListPackagesCommand : CliktCommand(
         help = "Omit all packages not matching all licenses given by this comma separated list of license identifiers."
     ).split(",").default(emptyList())
 
+    private val type by option(
+        "--type",
+        help = "Filter the output by type which can be either 'projects' or 'packages'."
+    ).convert { DataEntity.valueOf(it.uppercase()) }
```

Comment:
I'd recommend to reuse this code from `DownloadCommand`:

    .enum<DataEntity>().split(",").default(enumValues<DataEntity>().asList())

By setting a default value you do not have to handle the `null` case below in line 70 below.